tutorials/plot_background_filtering.py

```diff
@@ -483,15 +524,232 @@ def plot_signal(x, offset):
+#       to the filtering parameters.
+#    2. The validity of the assumption of no consistent evoked activity
+#       in the baseline period.
+#
+# We thus recommend carefully applying baseline correction and/or high-pass
+# values based on the characteristics of the data to be analyzed.
+#
+# .. _tut_filtering_in_python:
+#
+# Filtering defaults in MNE-Python
+# ================================
 # Most often, filtering in MNE-Python is done at the :class:`mne.io.Raw` level,
 # and thus :func:`mne.io.Raw.filter` is used. This function under the hood
 # (among other things) calls :func:`mne.filter.filter_data` to actually
-# filter the data.
+# filter the data, which by default applies a zero-phase FIR filter designed
+# using :func:`scipy.signal.firwin2`. In Widmann *et al.* 2015 [7]_, they
+# suggest a specific set of parameters to use for high-pass filtering,
+# including:
+#
+#     "... providing a transition bandwidth of 25% of the lower passband
+#     edge but, where possible, not lower than 2 Hz and otherwise the
+#     distance from the passband edge to the critical frequency.”
+#
+# In practice, this means that for each high-pass value ``l_freq`` or
+# low-pass value ``h_freq`` below, you would get this corresponding
+# ``l_trans_bandwidth`` or ``h_trans_bandwidth``, respectively,
+# if the sample rate were 100 Hz (i.e., Nyquist frequency of 50 Hz):
+#
+# +------------------+-------------------+-------------------+
+# | l_freq or h_freq | l_trans_bandwidth | h_trans_bandwidth |
+# +==================+===================+===================+
+# |             0.01 |              0.01 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              0.1 |               0.1 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              1.0 |               1.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              2.0 |               2.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              4.0 |               2.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |              8.0 |               2.0 |               2.0 |
+# +------------------+-------------------+-------------------+
+# |             10.0 |               2.5 |               2.5 |
+# +------------------+-------------------+-------------------+
+# |             20.0 |               5.0 |               5.0 |
+# +------------------+-------------------+-------------------+
+# |             40.0 |              10.0 |              10.0 |
+# +------------------+-------------------+-------------------+
+# |             45.0 |             11.25 |              5.0  |
+# +------------------+-------------------+-------------------+
+# |             48.0 |              12.0 |              2.0  |
+# +------------------+-------------------+-------------------+
```

Comment:
This table doesn't render properly on https://1891-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/auto_tutorials/plot_background_filtering.html